stable/airflow/templates/service-web.yaml

```diff
@@ -22,4 +22,7 @@ spec:
     - name: web
       protocol: TCP
       port: {{ .Values.airflow.service.externalPort | default 8080 }}
+      {{- if .Values.airflow.service.nodePort }}
```

Comment:
I have a consider on this, why we need to add a NodePort to the chart?   The NodePort should be random create by the controller. Also, the ingress can not transfer the `TCP` traffic, but I think `Istio` can help us do that.

Reply:
Hi. I don't know the intend behind this, but freezing the nodeport allows to access to the pod more easily knowning the port, without using an ingress.

I think no matter the use case one should always use an ingress/loadbalancer/whatever, but for debug I can understand the use case.

I would prefer however a solution such as:
```yaml
{{- if (and (eq .Values.service.type "NodePort") (not (empty .Values.service.nodePort.http))) }}
      nodePort: {{ .Values.service.nodePort.http }}
      {{- end }}
```

Reply:
Yes, I believe that can be made more sense.

Reply:
@thirtyseven can you update and update your PR pleae?

Reply:
> Hi. I don't know the intend behind this, but freezing the nodeport allows to access to the pod more easily knowning the port, without using an ingress.
> 
> I think no matter the use case one should always use an ingress/loadbalancer/whatever, but for debug I can understand the use case.

Yes, that's the intent of this PR.  We are using this chart in a k8s environment that does not currently support ingress controllers, but does have Consul DNS for service discovery.  

I'll make the changes as requested.